packages/nimble-components/src/table/specs/table-column-sort-hld.md

```diff
@@ -67,6 +89,31 @@ Each column that is sorted will have an appropriate icon displayed in the header
 
 Each column that is sorted ascending will have `aria-sort="ascending"` set on the header, and each column that is sorted descending will have `aria-sort="descending"` set on the header.
 
+### Interactive Sorting UX
```

Comment:
It'd be good to mention the keyboard interaction and accessibility questions like what role the focusable element will have. If that's too far out of scope to know yet then let's list it in the Out of Scope section above.

Reply:
Re-added Out of Scope section at the top, and listed those in that section. I think both are covered by separate open work items, so linked to those.